packages/client/src/generation/utils/buildNFTAnnotations.ts

```diff
@@ -30,8 +30,24 @@ export function buildNFTAnnotations(
     return ''
   }
 
+  // Add annotation for Netlify for a specific binaryTarget (depending on Node version and special env var)
   if (process.env.NETLIFY) {
-    platforms = ['rhel-openssl-1.0.x']
+    const isNodeMajor20OrUp = parseInt(process.versions.node.split('.')[0]) >= 20
+
+    // Netlify reads and changes the runtime version based on this env var
+    // https://docs.netlify.com/configure-builds/environment-variables/#netlify-configuration-variables
+    const awsRuntimeVersion = parseAWSNodejsRuntimeEnvVarVersion()
+    const isRuntimeEnvVar20OrUp = awsRuntimeVersion && awsRuntimeVersion >= 20
+    const isRuntimeEnvVar18OrDown = awsRuntimeVersion && awsRuntimeVersion <= 18
```

Comment:
Not critical: would be nice to have a shared function

Reply:
I'll merge this as is for now